src/Symfony/Component/HttpClient/Tests/HttplugClientTest.php

```diff
@@ -62,6 +96,32 @@ public function testNetworkException()
         $client->sendRequest($client->createRequest('GET', 'http://localhost:8058'));
     }
 
+    public function testAsyncNetworkException()
+    {
+        $client = new HttplugClient(new NativeHttpClient());
+
+        $promise = $client->sendAsyncRequest($client->createRequest('GET', 'http://localhost:8058'));
+        $successCallableCalled = false;
+        $failureCallableCalled = false;
+        $promise->then(function (ResponseInterface $response) use (&$callableCalled) {
```

Comment:
Shouldn't it be `$successCallableCalled`?

Reply:
good catch, fixed